local-volume/provisioner/pkg/common/common.go

```diff
@@ -130,20 +133,12 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 // GetVolumeConfigFromMountedConfigMap gets volume configuration from the mounted configmap volume,
 func GetVolumeConfigFromMountedConfigMap() (map[string]MountConfig, error) {
 	storageClassMapping := make(map[string]string)
-	files, err := ioutil.ReadDir(ProvisionerConfigPath)
+	fileContents, err := ioutil.ReadFile(path.Join(ProvisionerConfigPath, ProvisonerStorageClassConfigFile))
 	if err != nil {
+		glog.Infof("Could not read file: %s due to: %v", path.Join(ProvisionerConfigPath, ProvisonerStorageClassConfigFile), err)
 		return nil, err
 	}
-	for _, file := range files {
-		if !file.IsDir() {
-			fileContents, err := ioutil.ReadFile(path.Join(ProvisionerConfigPath, file.Name()))
-			if err != nil {
-				glog.Infof("Could not read file: %s due to: %v", path.Join(ProvisionerConfigPath, file.Name()), err)
-				continue
-			}
-			storageClassMapping[file.Name()] = string(fileContents)
-		}
-	}
+	storageClassMapping[ProvisonerStorageClassConfigFile] = string(fileContents)
 	return ConfigMapDataToVolumeConfig(storageClassMapping)
```

Comment:
This should be renamed to something like "StorageClassConfigMapDataToVolumeConfig"

Reply:
will do.